invisible_cities/core/core_functions_test.py

```diff
@@ -72,6 +80,32 @@ def test_in_range_right_shape(data):
     assert core.in_range(data, -1., 1.).shape == data.shape
 
 
+@given(sorted_unique_arrays)
+def test_in_range_left_open_interval(data):
+    output =core.in_range(data, minval=data[0], left="open")
+    assert not output[0]
+    assert all(output[1:])
+
+
+@given(sorted_unique_arrays)
+def test_in_range_right_open_interval(data):
+    output = core.in_range(data, maxval=data[-1], right="open")
+    assert not output[-1]
+    assert all(output[:-1])
+
+
+@given(sorted_unique_arrays)
+def test_in_range_left_close_interval(data):
+    output =core.in_range(data, minval=data[0], left="close")
+    assert all(output)
+
```

Comment:
Another missing space.

Also, this is the default behaviour and should have an open right side, shouldn't it? I understood this should mean the [-1] entry is false?

Reply:
It is the default behavior with left-closed and right-open. However the right doesn't matter because it's infinite (default value), therefore all elements must be within range. Also it should be the same as `all(core.in_range(data, minval=data[0], maxval=data[-1], left="close", right="close")`